ui/perfherder/alerts/FileBugModal.jsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Form,
+  FormGroup,
+  Input,
+  Label,
+  Button,
+  Modal,
+  ModalHeader,
+  ModalBody,
+  ModalFooter,
+  Col,
+  Row,
+} from 'reactstrap';
+import debounce from 'lodash/debounce';
+
+export default class FileBugModal extends React.Component {
+  // eslint-disable-next-line react/sort-comp
+  constructor(props) {
+    super(props);
+    this.state = {
+      inputValue: '',
+      invalidInput: false,
+      validated: false,
+    };
+  }
+
+  validateInput = debounce(() => {
+    const { inputValue } = this.state;
+    const regex = /^[1-9]+[0-9]*$/;
+    const updates = { validated: false, invalidInput: false };
+
+    if (!inputValue.length) {
+      updates.validated = true;
+      updates.invalidInput = false;
+    }
+
+    if (!inputValue.match(regex)) {
+      updates.invalidInput = true;
+    }
+
+    this.setState(updates);
+  }, 0);
```

Comment:
Because it takes one second after the culprit id is inputted to disable the `File bug` button, and if it's not valid (starts with 0 or contains non numeric characters) it will allow pressing file bug when it shouldn't. Also, the error message appears after one second, which seems a bit much to me.

Reply:
We could still add 150ms & still respect these conditions.